lib/src/folding/parsers/highlight.dart

```diff
@@ -2,10 +2,10 @@ import 'package:charcode/ascii.dart';
 import 'package:collection/collection.dart';
 import 'package:highlight/highlight_core.dart';
 
+import '../../../flutter_code_editor.dart';
```

Comment:
This file imports everything this package exports.
I try to only import the specific files we need but it is not a conscious decision.
Can you help me list the benefits of each pattern?

Reply:
Unfortunately, I don't have a strong opinion here either. But I also think, that importing specific files is good practice, because this should make it easier for the compiler. Fixed here